config/routes.rb

```diff
@@ -10,6 +10,8 @@
   resources :messages
   root to: 'visitors#index'
 
+  post 'help_request', to: 'application#help_request'
```

Comment:
create `help_requests_controller` . this logic shouldn't be in application controller